web/html/src/manager/virtualization/guests/console/guests-console.css

```diff
@@ -18,6 +18,14 @@
   margin: auto;
 }
 
+.canvas div {
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  color: #dcddde;
```

Comment:
Not for this PR but more in general: it would be nice if CSS rules/files are all in the `branding` folder and written in LESS as we do for the product style. It is fine to have a specific and separated file as it is for a specific feature, but with this detached CSS file, for instance, we are losing the power of changing color variable values and get it propagated to all the styles. Nothing to be fixed for now, but please keep it in mind before adding more rules here. :wink: